Source/src/main/java/CyberPunkReas/CovidSimulator/models/Restriction.java

```diff
@@ -1,31 +1,41 @@
-package CyberPunkReas.CovidSimulator.models;
+package Master.Design_Models;
 
 public class Restriction {
 
     private boolean curfew;
-    private boolean businessEssential;
-    private boolean businessNonEssential;
+    private boolean businessEssentialClosed;
+    private boolean businessNonEssentialClosed;
     private int timeRestriction;
-
+    private boolean mobility ;
+    
     public Restriction() {
         curfew = false;
-        businessEssential = false;
-        businessNonEssential = false;
+        businessEssentialClosed = false;
+        businessNonEssentialClosed = false;
         timeRestriction = 0;
+        mobility =false;
+    }
+    // Constructor for mockup
+    public Restriction(boolean curfew, boolean businessE,boolean businessNE,int timeRestriction, boolean mobility) {
+    	this.curfew = curfew;
+    	businessEssentialClosed = businessE;
+    	businessNonEssentialClosed = businessNE;
+    	this.timeRestriction= timeRestriction; 
+    	mobility = false;
     }
 
-    //Getters are created for RestrictionTest.java
+    //Getters are created in case they are needed for RestrictionTest.java
 
     public boolean isCurfew() {
         return curfew;
     }
 
     public boolean isBusinessEssential() {
```

Comment:
is the restriction BusinessEssential? That's what I understand here, but most likely you forgot to add `Closed`